app/workers/databases.php

```diff
@@ -1,10 +1,13 @@
 <?php
 
 use Appwrite\Event\Event;
+use Appwrite\Extend\Exception;
 use Appwrite\Messaging\Adapter\Realtime;
 use Appwrite\Resque\Worker;
 use Utopia\CLI\Console;
+use Utopia\Database\Database;
 use Utopia\Database\Document;
+use Utopia\Database\Helpers\ID;
```

Comment:
unused

Reply:
This one is used with `custom` a few places